kimchi/src/prover.rs

```diff
@@ -296,6 +307,7 @@ where
             lookup_context.sorted_coeffs = Some(sorted_coeffs);
             lookup_context.sorted_comm = Some(sorted_comm);
             lookup_context.sorted8 = Some(sorted8);
+            lookup_context.joint_lookup_table_d8 = Some(joint_lookup_table_d8);
```

Comment:
this time we store the computation of the combined table the first time we compute it